stubs/redis/redis/commands/core.pyi

```diff
@@ -187,9 +444,38 @@ class CoreCommands(Generic[_StrType]):
         offset: int | None = ...,
         num: int | None = ...,
     ) -> list[_StrType]: ...
+    @overload
+    def zrevrange(
+        self,
+        name: _Key,
+        start: int,
+        end: int,
+        *,
+        withscores: Literal[True],
```

Comment:
This doesn't handle setting `withscores` to `True` with a positional argument. `zrange()` doesn't handle that either.